### ISSUE TITLE
With stream adapter, $stack[0]->content is null, but with curl it is not

### ISSUE THREAD

**Issue by harikt** [auraphp/Aura.Http#29]
Hi Paul, 

I should give you some time before I open an issue for version 2 . This is just for future reference and testing the same.

When we use `stream` adapter the `$stack[0]->content`  is null. When changing to `curl` it works fine.

``` php
<?php
require __DIR__ . '/autoload.php';

$http_factory = new Aura\Http\HttpFactory();
$http = $http_factory->newInstance('stream');

$request = $http->newRequest();
$request->setUrl('https://api.github.com/orgs/auraphp/repos');
$request->setMethod(Aura\Http\Message\Request::METHOD_GET);
$request->headers->set('Accept', 'application/vnd.github.beta+json');
$request->headers->set('User-Agent', 'Mozilla');
$stack = $http->sendRequest($request);
$repos = json_decode($stack[0]->content);
foreach ($repos as $repo) {
    echo $repo->name . PHP_EOL;
}
```

**Comment by pmjones** [comment]
@harikt have you been able to figure out a fix for this?

**Comment by harikt** [comment]
I wasn't able to figure out :-/ .

**Comment by harikt** [comment]
I have been trying to understand / send a fix for the issue. But this seems hard to understand why github sends a 403 response only for the stream.

I have tried other urls, but the only issue I can see is from @github api. May be they have something special / checking in the headers which it is not set in the header.

**Comment by pmjones** [comment]
Fixed by 2adb99f (at last)
